dbt/targets.py

```diff
@@ -143,7 +143,7 @@ def context(self):
 
 target_map = {
```

Comment:
This is still used in two places:

```
(dbt) ✔ ~/git/dbt [adapters/initial|✔]
16:04 $ git grep dbt.targets
dbt/schema_tester.py:import dbt.targets
dbt/schema_tester.py:        return dbt.targets.get_target(target_cfg)
dbt/seeder.py:import dbt.targets
dbt/seeder.py:        self.target = dbt.targets.get_target(run_environment)
```

These two just don't support Snowflake. I just tested this and I got kind of a strange error:

```
✘-1 ~/git/hosted-test [master|✚ 1]
16:13 $ dbt seed --profile snowflake-test
Encountered an error while reading the project:
  ERROR Expected project configuration 'host' was not supplied
Did you set the correct --profile? Using: snowflake-test
Valid profiles:
 - hosted-test
 - snowflake-test
Encountered an error:
Could not run dbt
```

I'll work on that.